launcher/sdw-launcher.py

```diff
@@ -17,11 +38,24 @@ def main():
         # Logged.
         sys.exit(1)
     sdlog.info("Starting SecureDrop Launcher")
-    app = QtGui.QApplication(sys.argv)
-    form = UpdaterApp()
-    form.show()
-    sys.exit(app.exec_())
+
+    args = parse_argv(argv)
+
+    try:
```

Comment:
Untested, but perhaps something like `interval = int(getattr(args, 'skip_delta', DEFAULT_INTERVAL))` would help simplify the logic here